test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -199,7 +186,7 @@ def check_fc(existing_schemas):
     is_fc = True
     broken_ops = []
     for existing_schema in existing_schemas:
-        if allow_listed(existing_schema):
+        if allow_listed_with_deadline(existing_schema):
```

Comment:
I thought we were going to have a separate list for FC breaking allowlist? And why isn't the list without deadline checked here?

Reply:
I was gonna do that in a separate PR. In thi PR, I tried not to touch the fc part of the test. But yeah, i think it makes sense to add the one without deadline as well.